src/hooks/useFhirClient.tsx

```diff
@@ -12,9 +12,15 @@ type ResourceTypes = {
 };
 type ResourceType = Observation;
 
+type FhirCodes = {
+  system: string;
+  code: string;
+};
+
 type QueryParams = {
   resourceType: keyof ResourceTypes;
   pageSize?: number;
+  coding?: FhirCodes[];
```

Comment:
Let's use `Coding[]` here (from `fhir/r3`) instead, so that we don't need to create the `FhirCodes` type.